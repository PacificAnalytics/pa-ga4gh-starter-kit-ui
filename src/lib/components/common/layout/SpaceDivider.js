import React from 'react';
import { 
    Box,
    Divider
} from '@material-ui/core';

const SpaceDivider = () => {
    return (
        <Box py={4}>
            <Divider />
        </Box>
    );
}

export default SpaceDivider;
