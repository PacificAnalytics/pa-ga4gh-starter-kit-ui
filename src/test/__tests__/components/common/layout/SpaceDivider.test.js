import React from 'react';
import {render} from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import SpaceDivider from '../../../../../lib/components/common/layout/SpaceDivider';

test('<SpaceDivider />', () => {
    const {container} = render(<SpaceDivider />);
    expect(container.firstChild).toMatchSnapshot();
});