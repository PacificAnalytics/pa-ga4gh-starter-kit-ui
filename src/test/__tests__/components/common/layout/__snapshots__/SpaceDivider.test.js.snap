// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SpaceDivider /> 1`] = `
<div
  class="MuiBox-root MuiBox-root-1"
>
  <hr
    class="MuiDivider-root"
  />
</div>
`;
